Bad atomic field offset on i386

I currently run into the following issue 
```
> go-fuzz
92
panic: bad atomic field offset

goroutine 1 [running]:
main.init.1()
	/home/kali/go/src/github.com/dvyukov/go-fuzz/go-fuzz/testee.go:64 +0x50
```
go version: `go version go1.14.9 linux/386`

--- a/go-fuzz/testee.go
+++ b/go-fuzz/testee.go
@@ -31,8 +31,8 @@ type Testee struct {
 	stdoutPipe  *os.File
 	writebuf    [9]byte  // reusable write buffer
 	resbuf      [24]byte // reusable results buffer
-	execs       int
 	startTime   int64
+	execs       int
 	outputC     chan []byte
 	downC       chan bool
 	down        bool

